test/gtest/uct/uct_test.h

```diff
@@ -150,19 +150,25 @@ class uct_test : public testing::TestWithParam<const resource*>,
         void destroy_eps();
         void connect(unsigned index, entity& other, unsigned other_index);
         void connect(unsigned index, entity& other, unsigned other_index,
-                     const ucs::sock_addr_storage &remote_addr);
+                     const ucs::sock_addr_storage &remote_addr,
+                     uct_ep_client_connect_cb_t connect_cb,
+                     uct_ep_disconnect_cb_t disconnect_cb,
+                     void *user_data);
         void connect_to_iface(unsigned index, entity& other);
         void connect_to_ep(unsigned index, entity& other,
                            unsigned other_index);
         void connect_to_sockaddr(unsigned index, entity& other,
```

Comment:
maybe split for 2 funcs: `connect_to_listener_sockaddr` and `connect_to_iface_sockaddr`?

Reply:
IMHO it's not necessary to split since the old flow will be deprecated anyway so we'll be left with one function, as it is now.

Reply:
IMHO separate func easier delete than cleanup common one, but I'm fine with any solution